CMakeLists.txt

```diff
@@ -36,10 +36,26 @@ set(NULL_ENTROPY_WARNING "${WARNING_BORDER}"
                          "${NULL_ENTROPY_WARN_L3}"
                          "${WARNING_BORDER}")
 
-find_package(PythonInterp)
```

Comment:
Is it intentional to remove `find_package(PythonInterp)`?

Reply:
no,  fixed.